### ISSUE TITLE
Transcription fails while log states completed

### ISSUE THREAD

**Issue by JuergenFleiss** [JuergenFleiss/aTrain#53]
I can confirm that this issue is reproducible with multiple files.

I’ve been unable to identify the root cause. Initially, I attempted to use a single 5-hour-long audio file, which exhibited the same behavior. To troubleshoot, I split the file into multiple 19-minute segments. While some of these files were successfully transcribed, about one-third consistently fail, and I’m unable to transcribe them at all.

All the files are similar in size. For the successful ones, the operation completes in 1–3 minutes. However, for the problematic files, the process gets stuck, though the logs always indicate the status as "completed."

@JuergenFleiss, would you like me to run additional tests or log a new issue for this?

_Originally posted by @OhsawaSenpai in https://github.com/JuergenFleiss/aTrain/issues/41#issuecomment-2530876951_

**Comment by JuergenFleiss** [comment]
Hi @OhsawaSenpai 

I put it in a new issue, as I think it is not related to the stuck UI bug where the transcription successfully completes in the background.

Any chance you could privately share the file with me so I can try to reproduce it? I can also offer you guest access to upload it to our on premise hosted university file sharing so that the data is secure (https://share.uni-graz.at/).

**Comment by OhsawaSenpai** [comment]
Hi @JuergenFleiss 

Thank you for following up on this.

I have a link that I can share with you with the original file, and also the split files.
Would you mind letting me know what is the best way for me to share it?

**Comment by JuergenFleiss** [comment]
best would be bandas@uni-graz.at

Thx, that helps a lot

**Comment by OhsawaSenpai** [comment]
@JuergenFleiss email with the link has been sent, thank you!

**Comment by Petertttt** [comment]
Here also an often occuring issue with the user interface getting stuck during the proces of transcribing a file. As soon as my GPU-load indicator drops, the transcript is outputted, and complete.

**Comment by JuergenFleiss** [comment]
Yes, we think it should be solved; the one test file where we could reproduce the error now completes successfully.

**Comment by OhsawaSenpai** [comment]
I will test this as soon as I'm able to update aTrain to the latest version (it is not showing up the option to update in the Windows Store, still stuck with 1.2.0).

**Comment by JuergenFleiss** [comment]
There where still some issues in 1.2.1; @ArminHaberl  has now fixed those in the 1.2.2 branches of atrain and atrain_core

We will update Windows Store Version directly to 1.2.2 when no new issues arise in testing.

**Comment by OhsawaSenpai** [comment]
It seems that 1.2.2 is now available in Windows Store.

I will perform some tests with some audio files that I have that were giving issues before and check if it works properly now.

Thank you!

**Comment by JuergenFleiss** [comment]
Thanks for testing

**Comment by OhsawaSenpai** [comment]
I have just tried and aTrain is now crashing for me.

I've tried with
- Model: large-v3-turbo
- Language: automatic and Portuguese
- Multi-speaker: on and off
- GPU: in8, float16 and float32
- No prompt; no CPU test;

Is there anyway I can help checking this issue, like could I share some logs with you?

**Comment by JuergenFleiss** [comment]
Do you get any error message?

**Comment by OhsawaSenpai** [comment]
No error message, the app simply crashes suddenly.

**Comment by OhsawaSenpai** [comment]
Btw, I have just tried with CPU instead of GPU and it worked fine.

However, prior to the current version, I've always used GPU without any issues. 

Is it possible to somehow downgrade the version on my end for testing purposes?

**Comment by JuergenFleiss** [comment]
You can download the old installers here:
https://business-analytics.uni-graz.at/de/forschung/atrain/download/

**Comment by OhsawaSenpai** [comment]
Thanks!

I will test and provide some feedback.

**Comment by JuergenFleiss** [comment]
Ok, can reproduce it with the newest version on Windows and GPU. This makes no sense...

We will hopefully have a fix soon

**Comment by OhsawaSenpai** [comment]
I've also tested with the older files that before I was not able to transcribe, and with CPU it seems they work fine as well.

**Comment by JuergenFleiss** [comment]
Ok, we think we have found the problem but will do some more testing with different files and systems. Confident that we can upload a fixed version for GPU to the Windows store this week.

**Comment by OhsawaSenpai** [comment]
That's great! Let me know once it is done, and I'll test on my end and share the results with you.

**Comment by bullkelp** [comment]
Experiencing the exact same issue. Watching this for updates.

**Comment by dacaz5** [comment]
I'm also having this issue. Is there a fix for this coming soon?

**Comment by JuergenFleiss** [comment]
Hi, yes we went to back 1.2.0 (but could not in the microsoft store) due to problems with 1.2.3

This brought back the frozen ui bug. The newest version 1.2.4 should finally take care of all the issues at once.

**Comment by JuergenFleiss** [comment]
Fixed in https://github.com/JuergenFleiss/aTrain/releases/tag/v1.3.0
